src/DataStructures.jl

```diff
@@ -94,4 +94,6 @@ module DataStructures
     export status
     export deref_key, deref_value, deref, advance, regress
 
+    @deprecate iter(s::Stack) collect(s)
+    @deprecate iter(q::Queue) collect(q)
```

Comment:
I wonder... `collect` does do essentially what `iter` used to do, so if someone used `iter` expecting an array, then this is correct.  However, if they're using `iter` to iterate over the stack or queue, then `collect(s)` and `collect(q)` should just be `s` and `q`...